barretenberg/cpp/src/barretenberg/dsl/acir_format/serde/acir.hpp

```diff
@@ -1234,7 +1234,7 @@ struct AssertionPayload {
     };
 
     struct Dynamic {
-        std::tuple<uint64_t, std::vector<Program::ExpressionOrMemory>> value;
```

Comment:
Can you revert this change and open a version of the PR in the main noir repo? This will avoid us not being able to merge this until the next release and then getting a bunch of nasty merge conflicts.

We can truncate the error selectors in brillig for the time being and then update the selector size in a separate PR after the rest of the changes are synced from Noir to aztec-packages.